### PR TITLE
Make form grid gutter width configurable

### PR DESCRIPTION
**Pull request by jacobmllr95**
This PR makes the form grid gutter width configurable by the `$form-grid-gutter-width` variable.

### CODE REVIEW

File: scss/_variables.scss [hunk 1]
```diff
@@ -499,6 +499,8 @@ $input-transition:                      border-color .15s ease-in-out, box-shado
 
 $form-text-margin-top:                  .25rem !default;
 
+$form-grid-gutter-width:                10px !default;
```
Comment (MartijnCuppens) [inline scss/_variables.scss]: I would move this line to where `$form-group-margin-bottom` is set, those variables are a bit related.

File: scss/_forms.scss [hunk 2]
```diff
@@ -190,13 +190,13 @@ textarea.form-control {
...
-  margin-right: -5px;
-  margin-left: -5px;
+  margin-right: -($form-grid-gutter-width / 2);
+  margin-left: -($form-grid-gutter-width / 2);
 
   > .col,
   > [class*="col-"] {
-    padding-right: 5px;
-    padding-left: 5px;
+    padding-right: ($form-grid-gutter-width / 2);
```
Comment (MartijnCuppens) [inline scss/_forms.scss]: I don't think the brackets are needed here, can you check if you can remove them?

Comment (jackmu95) [reply scss/_forms.scss]: I don't think so too, but they are also present in [here](https://github.com/twbs/bootstrap/blob/v4-dev/scss/mixins/_grid-framework.scss#L11).

Comment (MartijnCuppens) [reply scss/_forms.scss]: Good point. I just added #27768 to remove those redundant brackets. You can also remove it here.

**Review by MartijnCuppens — APPROVED** [review]
LGTM!

**Pull request merged**